src/algoliasearch.helper.js

```diff
@@ -343,6 +343,61 @@ AlgoliaSearchHelper.prototype.getCurrentPage = function() {
   return this.state.page;
 };
 
+/**
+ * Get the list of refinements for a give facet.
+ * @param {string} facetName
+ * @return {Refinement[]} All Refinement contains a value, and a type. Numeric also contains operators.
+ */
+AlgoliaSearchHelper.prototype.getRefinements = function( facetName ) {
+  var refinements = [];
+
+  if( this.state.isConjunctiveFacet( facetName ) ) {
+    var conjRefinements = this.state.getConjunctiveRefinements( facetName );
+    if( conjRefinements ) {
+      forEach( conjRefinements, function( r ) {
+        refinements.push( {
+          value : r,
+          type : "conjunctive"
+        } );
+      } );
+    }
+  }
+  else if( this.state.isDisjunctiveFacet( facetName ) ) {
+    var disjRefinements = this.state.getDisjunctiveRefinements( facetName );
+    if( disjRefinements ) {
+      forEach( disjRefinements, function( r ) {
+        refinements.push( {
+          value : r,
+          type : "disjunctive"
+        } );
+      } );
+    }
+  }
+
+  var excludeRefinements = this.state.getExcludeRefinements( facetName );
+  if( excludeRefinements ) {
+    forEach( excludeRefinements, function( r ) {
+      refinements.push( {
+        value : r,
+        type : "exclude"
+      } );
+    } );
+  }
+
+  var numericRefinements = this.state.getNumericRefinements( facetName );
+  if( numericRefinements ) {
+    forEach( numericRefinements, function( value, operator ) {
+      refinements.push( {
+        value : value,
+        operator : operator,
+        type : "numeric"
+      } );
+    } );
+  }
+
+  return refinements;
+};
+
 ///////////// PRIVATE
```

Comment:
Oh OK, so here it returns an array even if it doesn't exist. Could make sense to only have it here!